plugins/gossip/plugin.go

```diff
@@ -46,7 +42,7 @@ func configure(*node.Plugin) {
 
 func run(*node.Plugin) {
 	if err := daemon.BackgroundWorker(PluginName, start, shutdown.PriorityGossip); err != nil {
-		log.Panicf("Failed to start as daemon: %s", err)
+		plugin.Logger().Panicf("Failed to start as daemon: %s", err)
```

Comment:
Since it always should be initialized I think this is the easier option

Reply:
Still, that seems very strange to me as we even have this (otherwise unused) `plugin *node.Plugin` parameter two lines above.
However, if you don't like using that how about using `Plugin()` to indicate that this will always be initialized?